### PR TITLE
misc: add cjs path for logger

### PR DESCRIPTION
**Pull request by adamraine**
We need to do a patch release to restore cjs since the update today broke some folks downstream.

I packaged this logger and verified that chrome launcher passed all it's tests with it.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,7 +7,7 @@ npm-debug.log
 .tmp_env
 .tgz
 
-/dist
+**/dist
```
Comment (connorjclark) [inline .gitignore:10]: I feel like this / above can be just `dist/`?

**Review by connorjclark — APPROVED** [review]

**Comment by brendankenny** [comment]
Does this mean there will be two versions used in a typical LH run?

**Comment by adamraine** [comment]
> Does this mean there will be two versions used in a typical LH run?

That may have already been the case w/ chrome launcher requesting version 1.0.0 and lighthouse requesting version 1.3.0 (soon to be 1.4.1)

**Comment by brendankenny** [comment]
> That may have already been the case w/ chrome launcher requesting version 1.0.0 and lighthouse requesting version 1.3.0 (soon to be 1.4.1)

Sure, but that's always the case. chrome-launcher is requesting `^1.0.0` which is compatible with the `^1.4.1` added in #15082, so if yarn didn't suck with updates that PR would still have a single version like it does today. This PR adds two versions even if lighthouse-logger is resolved to a single package.

It would be good to have a path out of this situation (e.g. update chrome-launcher).

**Comment by connorjclark** [comment]
All the state is stored in `marky`, nothing in Logger, so aren't things shared already?

**Comment by brendankenny** [comment]
> All the state is stored in marky and debug, nothing in Logger

Today it is!

We control all the pieces here, so there's no reason we can't make it simple again now that the shipping version is working.

**Pull request merged**